client/src/BaseFilterControls.ts

```diff
@@ -22,8 +21,9 @@ export interface AnnotationWithContext<T extends Track | Group> {
 }
 
 export interface FilterControlsParams<T extends Track | Group> {
-  store: BaseAnnotationStore<T>;
+  sorted: Ref<T[]>;
   markChangesPending: () => void;
+  remove: (id: AnnotationId) => void;
```

Comment:
This moves back toward the `useFilterControls` style where we passed individual bits of state and handlers.

I understand why it's necessary, I just want to avoid going down the path where all our constructor functions end up looking like the composition function params used to with half a dozen arguments each.

Nothing to do here, just something to think about.